specs/capella/beacon-chain.md

```diff
@@ -381,19 +280,54 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
     process_sync_aggregate(state, block.body.sync_aggregate)
 ```
 
+#### New `get_expected_withdrawals`
+
+```python
+def get_expected_withdrawals(state: BeaconState) -> Sequence[Withdrawal]:
+    epoch = get_current_epoch(state)
+    withdrawal_index = state.next_withdrawal_index
+    index = ValidatorIndex((state.last_withdrawal_validator_index + 1) % len(state.validators))
+    ret: List[Withdrawal] = []
+    probed = 0
+    while (len(ret) < MAX_WITHDRAWALS_PER_PAYLOAD) and (probed < len(state.validators)):
```

Comment:
would be nice to rewrite this as a `for` over `len(state.validators)` and break when we have `MAX_WITHDRAWALS_PER_PAYLOAD`